modules/health_quest/app/services/health_quest/patient_generated_data/options_builder.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module PatientGeneratedData
+    class OptionsBuilder
```

Comment:
Please add class-level and public method documentation via [YARDoc](https://yardoc.org/guides/index.html)

Reply:
Done!